rand_core/src/lib.rs

```diff
@@ -299,6 +299,12 @@ pub trait SeedableRng: Sized {
     /// reference implementation of the PRNG given equivalent seed; if necessary
     /// another constructor replicating behaviour from a reference
     /// implementation can be added.
+    ///
+    /// `from_seed` should never panic. Some PRNGs cannot use certain seeds, for
```

Comment:
It's not clear that this is a recommendation for implementations and outside of our control. Would it be better to start with "PRNG implementations" as the two advisory notices above?